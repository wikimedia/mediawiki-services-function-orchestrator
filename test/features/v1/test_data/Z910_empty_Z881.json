{
	"Z1K1": {
		"Z1K1": "Z9",
		"Z9K1": "Z7"
	},
	"Z7K1": {
		"Z1K1": {
			"Z1K1": "Z9",
			"Z9K1": "Z8"
		},
		"Z8K1": [
			{
				"Z1K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z17"
				},
				"Z17K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z1"
				},
				"Z17K2": {
					"Z1K1": "Z6",
					"Z6K1": "Z810K1"
				},
				"Z17K3": {
					"Z1K1": {
						"Z1K1": "Z9",
						"Z9K1": "Z12"
					},
					"Z12K1": []
				}
			},
			{
				"Z1K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z17"
				},
				"Z17K1": {
					"Z1K1": "Z7",
					"Z7K1": "Z881",
					"Z881K1": "Z6"
				},
				"Z17K2": {
					"Z1K1": "Z6",
					"Z6K1": "Z810K2"
				},
				"Z17K3": {
					"Z1K1": {
						"Z1K1": "Z9",
						"Z9K1": "Z12"
					},
					"Z12K1": []
				}
			}
		],
		"Z8K2": {
			"Z1K1": "Z9",
			"Z9K1": "Z1"
		},
		"Z8K3": [],
		"Z8K4": [
			{
				"Z1K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z14"
				},
				"Z14K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z810"
				},
				"Z14K4": {
					"Z1K1": "Z6",
					"Z6K1": "Z910"
				}
			}
		],
		"Z8K5": {
			"Z1K1": "Z9",
			"Z9K1": "Z810"
		}
	},
	"Z810K1": {
		"Z1K1": "Z6",
		"Z6K1": "Just cutting in line."
	},
	"Z810K2": {
		"Z1K1": {
			"Z1K1": "Z4",
			"Z4K1": {
				"Z1K1": "Z7",
				"Z7K1": "Z881",
				"Z881K1": "Z6"
			},
			"Z4K2": [
				{
					"Z1K1": "Z3",
					"Z3K1": "Z6",
					"Z3K2": {
						"Z1K1": "Z6",
						"Z6K1": "K1"
					},
					"Z3K3": {
						"Z1K1": "Z12",
						"Z12K1": []
					}
				},
				{
					"Z1K1": "Z3",
					"Z3K1": {
						"Z1K1": "Z7",
						"Z7K1": "Z881",
						"Z881K1": "Z6"
					},
					"Z3K2": {
						"Z1K1": "Z6",
						"Z6K1": "K2"
					},
					"Z3K3": {
						"Z1K1": "Z12",
						"Z12K1": []
					}
				}
			],
			"Z4K3": "Z110"
		}
	}
}
