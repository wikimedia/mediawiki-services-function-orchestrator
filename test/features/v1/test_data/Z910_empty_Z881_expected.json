{
	"Z1K1": {
		"Z1K1": "Z9",
		"Z9K1": "Z10"
	},
	"Z10K1": {
		"Z1K1": "Z6",
		"Z6K1": "Just cutting in line."
	},
	"Z10K2": {
		"Z1K1": {
			"Z1K1": "Z9",
			"Z9K1": "Z10"
		}
	}
}
